sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseExportDto.java

```diff
@@ -643,6 +645,16 @@ public String getInternalToken() {
 	}
 
 	@Order(7)
+	@ExportTarget(caseExportTypes = {
+		CaseExportType.CASE_SURVEILLANCE,
+		CaseExportType.CASE_MANAGEMENT })
+	@ExportProperty(CaseDataDto.CASE_REFERENCE_NUMBER)
+	@ExportGroup(ExportGroupType.CORE)
+	public String getCaseReferenceNumber() {
+		return caseReferenceNumber;
```

Comment:
I think that caseReference should not be displayed in the export files (based on the acceptance criteria "Case Reference is only shown when self reporting feature is enabled") when FeatureType.SELF_REPORTING is disabled.

Currently, when  FeatureType.SELF_REPORTING is disabled, the column will not appear in the grid as it was requested ( Contacts and Cases) but will appear in the Detailed exports for Cases for Contacts.

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/6f5d9ec4-c01a-4233-80b7-b53c1fad6fb9)
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/f5db46d1-3004-4d92-9f33-ac44938739ce)

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/6fcd368b-eb99-4f8c-92e2-0a49e124edf4)